Dev/src/components/StudentCard.js

```diff
@@ -79,17 +87,17 @@ StudentCard.propTypes = {
 const styles = StyleSheet.create({
     cardStyle: {
         flexDirection: 'row',
-        marginRight: 7,
-        height: 100,
+        marginRight: Dimensions.get('window').width * 0.017,
+        height: Dimensions.get('window').height * 0.112,
         alignItems: 'center',
-        marginLeft: 7,
-        marginTop: 10,
+        marginLeft: Dimensions.get('window').width * 0.017,
+        marginTop: Dimensions.get('window').height * 0.01,
         fontFamily: 'Montserrat-Regular',
     },
     removeStudentStyle: {
         flexDirection: 'row',
         justifyContent: 'center',
-        marginRight: 20,
+        marginRight: Dimensions.get('window').width * 0.05,

```

Comment:
It is better to declare 2 consts: windowWidth and windowHeight and assign to them the value of Dimensions.get('window').width and .height, and then reference the consts in the style instead of making the function calls everytime. Consts are free, function calls are not.